network.sh

```diff
@@ -151,3 +166,19 @@ else
   export BOOTSTRAP_PROVISIONING_IP=${BOOTSTRAP_PROVISIONING_IP:-$(nth_ip $PROVISIONING_NETWORK 2)}
   export CLUSTER_PROVISIONING_IP=${CLUSTER_PROVISIONING_IP:-$(nth_ip $PROVISIONING_NETWORK 3)}
 fi
+
+# Proxy related configuration
+if  [[ ! -z "$INSTALLER_PROXY" ]]; then
+  export EXT_SUBNET=${EXTERNAL_SUBNET_V6}
+  if [[ "$IP_STACK" = "v4" ]]; then
+    EXT_SUBNET=${EXTERNAL_SUBNET_V4}
+  fi
+
+  HTTP_PROXY=http://$(wrap_if_ipv6 ${PROVISIONING_HOST_EXTERNAL_IP}):${INSTALLER_PROXY_PORT}
+  HTTPS_PROXY=http://$(wrap_if_ipv6 ${PROVISIONING_HOST_EXTERNAL_IP}):${INSTALLER_PROXY_PORT}
+  NO_PROXY=${PROVISIONING_NETWORK},9999,${EXT_SUBNET}
+
+  if [[ "$PROVISIONING_NETWORK_PROFILE" != "Disabled" ]]; then
+    NO_PROXY=${EXT_SUBNET},9999
```

Comment:
I think this conditional is inverted, e.g we should omit `PROVISIONING_NETWORK` when `PROVISIONING_NETWORK_PROFILE` *is* `Disabled`?

Reply:
Oh, yes, good catch @hardys thanks!